### ISSUE TITLE
Using method later()

### ISSUE THREAD

**Issue by igibioy999** [iwasherefirst2/Laravel-MultiMail#30]
Hi there,
will it be possible to use the method later() to queue emails for later?
If not, is there a workaround how this can be done with this package?
Thanks a lot!

**Comment by niels-numbers** [comment]
Its not implemented now, but it should only be a few lines of code. Just take a look how queue() is implemented in PendingMail & MultiMail, we would need a later method which is basically the same, just delaying the dispatched job. Could you create a MR for that?

**Comment by igibioy999** [comment]
yes, I managed to implement it in the way as you described it. What is MR? :)

**Comment by niels-numbers** [comment]
@igibioy999 that is great! With MR I meant if you could create a Merge-Request, so I can add the later method to this package
